source/server/connection_handler_impl.h

```diff
@@ -86,17 +86,50 @@ class ConnectionHandlerImpl : public Network::TcpConnectionHandler,
     UdpListenerCallbacksOptRef udpListener();
     Network::InternalListenerOptRef internalListener();
   };
+
+  struct ActiveListenerDetails {
+    std::vector<std::shared_ptr<PerAddressActiveListenerDetails>> per_address_details_list_;
+
+    using ListenerMethodFn = std::function<void(Network::ConnectionHandler::ActiveListener&)>;
+
+    void invokeListenerMethod(ListenerMethodFn fn) {
```

Comment:
Please add some comments to public methods (helps when looking with an IDE and trying to understand what this is doing)

Reply:
got it